src/infrastructure/persistence/PsqlPermissionRepository.ts

```diff
@@ -0,0 +1,36 @@
+import { PermissionRepository } from '../../domain/model/authentication/PermissionRepository';
+import { Permission } from '../../domain/model/authentication/Permission';
+import knex from 'knex';
+
+export class PsqlPermissionRepository implements PermissionRepository {
+  constructor(private db: knex) {}
+
+  async findByName(name: string): Promise<Permission | null> {
+    const permissionRow: any = await this.db('permission')
+      .select(['name', 'creation_time'])
```

Comment:
as creationTime?

Reply:
Hmm.. Might have to update ALL repos if we do this 🤔